### ISSUE TITLE
Outbound Internet Protection

### ISSUE THREAD

**Issue by aumohr** [GoogleCloudPlatform/cloud-foundation-fabric#2021]
**Describe the bug**
In the current configuration, all systems, connected to the shared VPCs, have outbound Internet access through the Cloud NAT. By default, outbound Internet access should not be possible but must be explicitly enabled.

**Rational**
Creating a catch-all deny firewall rule for all egress traffic mitigates data exfiltration risks, if the VPC is directly connected to the Internet.

**Recommendation**
1) [...] Create a firewall rule with the lowest priority that blocks all outbound traffic for all protocols and ports. In combination with the default behavior of Compute Engine instances (namely, to deny all inbound traffic), this rule locks down your Compute Engine instances. 
[...]
3) When you enable Google APIs, you need to include Google IP addresses while creating higher-priority firewall rules. For example, if set up to use restricted.googleapis.com or [private.googleapis.com](http://private.googleapis.com/) for VPC-SC, you need to open up egress TCP 443 to 199.36.153.4/30 or 199.36.153.8/30.
[...]

**Expected behavior**
If CloudNAT is deployed, it should automatically include a deny all egress firewall rule. If CloudNAT is commented out, it should include a prepared firewall rule to use in conjunction with CloudNAT.

**Comment by ludoo** [comment]
We discussed this differently: we don't want to introduce egress rules as they are a source of additional complexity which is not always needed.

The change we discussed was to not create cloud nat by default. We should track here what we actually discuss, not the CSPR recommendations which are not always applicable to FAST, which is designed for quick deployment and incremental security.

**Comment by aumohr** [comment]
Agree, this is what I tried to describe in Expected behavior. "Rational" and "Recommendation" are indeed just the related extracts of CSPR

Not to create cloud nat (e.g. by uncommenting it) and add the warning to include the egress fw rule if used is ok. Sorry if the text was confusing.
